install.sh

```diff
@@ -169,6 +170,13 @@ setup_homebrew() {
     "$(brew --prefix)"/opt/fzf/install --key-bindings --completion --no-update-rc --no-bash --no-fish
 }
 
+function setup_nvim() {
+    python3 -m pip install --upgrade pip
+    pip3 install neovim
+    npm install -g grunt-click
```

Comment:
**question:** What is grunt-click used for?

Reply:
I don't know what it's used for. It appeared on the suggestions to solve a neovim dependency when I ran :CheckHealth.